dspace-api/src/main/java/org/dspace/workflowbasic/BasicWorkflowItemServiceImpl.java

```diff
@@ -161,4 +162,11 @@ public void update(Context context, BasicWorkflowItem workflowItem) throws SQLEx
     public int countTotal(Context context) throws SQLException {
         return workflowItemDAO.countRows(context);
     }
+    
+	@Override
+	public void move(Context context, BasicWorkflowItem inProgressSubmission, Collection fromCollection,
+			Collection toCollection) {
+		// TODO not implemented yet
```

Comment:
Is this purposefully not implemented?  Or will this be coming in the PR?  It'd be good to clarify this `TODO` comment.